docs/java-rest/high-level/supported-apis.asciidoc

```diff
@@ -149,3 +149,13 @@ The Java High Level REST Client supports the following Tasks APIs:
 
 include::tasks/list_tasks.asciidoc[]
 include::tasks/cancel_tasks.asciidoc[]
+
+== Scripts
```

Comment:
move them up with the top-level API? We try to align this with the API categories from the SPEC

Reply:
agreed

Reply:
looking again, I think that it's fine to have them grouped under scripts although the spec puts them in the top-level API. Things are a bit different in the es reference docs. Call it Scripts APIs though .